actioncable/app/assets/javascripts/action_cable/consumer.coffee

```diff
@@ -22,6 +33,12 @@ class ActionCable.Consumer
   send: (data) ->
     @connection.send(data)
 
+  start: ->
+    @connection.open()
+
+  stop: ->
+    @connection.close(true)
```

Comment:
Another thing to note is, we're adding extra terminology here. On top of `close` and `open` and the `connected` and `disconnected` callbacks, we now have `start` and `stop`. Since (dis)connect is generally what end-users will see that might be better than start & stop. E.g. `App.cable.connect()` over `App.cable.start()`